src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Xproj/MigrateXprojProjectFactory.cs

```diff
@@ -197,6 +210,23 @@ internal bool BackupProject(string backupLocation, string xprojLocation, string
             return (report.OutputMSBuildProject, report.Succeeded);
         }
 
+        internal void CleanupXproj(string projectLocation, string projectName)
+        {
+            // Clean up the .xproj, the project.json, the project.lock.json, and the xproj.user
+            _fileSystem.RemoveFile(Path.Combine(projectLocation, $"{projectName}.xproj"));
```

Comment:
Question: Why catch exceptions for removing lock.json and .user but not the others?